source/includes/v2/_assets.md

```diff
@@ -0,0 +1,110 @@
+# Assets
+
+Sirius provides the set of crypto assets available to use.
+You can get notifications about a deposit or initiate a withdrawal in any asset presented in this set.
+The asset has a generic model which represents a native assets an tokens.
```

Comment:
`The asset has a generic model which represents a native assets an tokens.` -> `The asset is a generic model which represents a native asset, fungible token or a non-fungible token.`?